hardware_interface/include/hardware_interface/types/hardware_interface_type_values.hpp

```diff
@@ -17,8 +17,13 @@
 
 namespace hardware_interface
 {
+/// Constant defining position interface
 constexpr const auto HW_IF_POSITION = "position";
+/// Constant defining velocity interface
 constexpr const auto HW_IF_VELOCITY = "velocity";
+/// Constant defining acceleration interface
+constexpr const auto HW_IF_EFFORT = "acceleration";
```

Comment:
this is when you forget to save the file...
```suggestion
constexpr const auto HW_IF_ACCELERATION = "acceleration";
```